src/main/java/software/sigma/internship/service/impl/TestServiceImpl.java

```diff
@@ -142,4 +160,10 @@ private List<Answer> mapAnswers(QuestionDto question, Question entityQuestion) {
                 }).collect(Collectors.toList());
     }
 
+
+    @Scheduled(cron = "0 0 0 ? * SUN")
+    @CacheEvict(cacheNames = {"tests", "testsByTeacher", "allTests"}, allEntries = true)
```

Comment:
Maybe it would be better move scheduler to different class

Reply:
Maybe :) I think it will be right to create a separate class with all scheduled methods